src/main/c/sslcontext.c

```diff
@@ -1204,11 +1203,10 @@ TCN_IMPLEMENT_CALL(void, SSLContext, setCertVerifyCallback)(TCN_STDARGS, jlong c
         if (method == NULL) {
             return;
         }
-        ver = malloc(sizeof(struct cert_verifier));
-        ver->verifier = (*e)->NewGlobalRef(e, verifier);
-        ver->method = method;
+        c->verifier = (*e)->NewGlobalRef(e, verifier);
+        c->verifier_method = method;
```

Comment:
What happens if a user calls `setCertVerifyCallback()` multiple times? Shouldn't we release the old callback?